Frameworks/UIKit/UIView.mm

```diff
@@ -162,14 +170,29 @@ - (bool)_processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEven
         }
     }
 
-    viewDepth = 0;
-
     g_curGesturesDict = [NSMutableDictionary new];
 
-    UIGestureRecognizer* recognizers[128];
+    const static int MAXIMUM_GESTURE_ALLOW = 128;
+    const static int MAXIMUM_DMGESTURE_ALLOW = 16;
 
+    UIGestureRecognizer* recognizers[MAXIMUM_GESTURE_ALLOW];
+    UIGestureRecognizer* dmrecognizers[MAXIMUM_DMGESTURE_ALLOW];
```

Comment:
>dm [](start = 25, length = 2)

more use of 'dm' - could definitely use more descriptive names across all of these files/file names/etc.